plugins/k8saudit-gke/hack/falco.yaml

```diff
@@ -0,0 +1,77 @@
+base_syscalls:
+  custom_set: []
+  repair: false
+buffered_outputs: false
+file_output:
+  enabled: false
+grpc:
+  bind_address: unix:///run/falco/falco.sock
+  enabled: false
+  threadiness: 0
+grpc_output:
+  enabled: false
+http_output:
+  ca_bundle: ""
+  ca_cert: ""
+  ca_path: /etc/ssl/certs
+  enabled: false
+  insecure: false
+  url: "http://127.0.0.1:2801/"
+  user_agent: falcosecurity/falco
+json_include_output_property: true
+json_include_tags_property: true
+json_output: true
+libs_logger:
+  enabled: false
+load_plugins: [ k8saudit-gke, json ]
+log_level: info
+log_stderr: true
+log_syslog: true
+metadata_download:
+  chunk_wait_us: 1000
+  max_mb: 100
+  watch_freq_sec: 1
+metrics:
+  convert_memory_to_mb: true
+  enabled: false
+  include_empty_values: false
+  interval: 5m
+  kernel_event_counters_enabled: true
+  libbpf_stats_enabled: true
+  output_rule: true
+  resource_utilization_enabled: true
+modern_bpf:
+  cpus_for_each_syscall_buffer: 2
+output_timeout: 2000
+plugins:
+- init_config:
+    project_id: "te000-589aa8c5b4314ba934a6"
+    fetch_cluster_metadata: false
+    max_event_size: 8000000
+  library_path: libk8saudit-gke.so
+  name: k8saudit-gke
+  # open_params: "file://test/pods_create.json"
+  open_params: "falco-gke-audit-sub"
+- init_config: ""
+  library_path: libjson.so
+  name: json
+priority: debug
+program_output:
+  enabled: false
+rules_file:
+- /etc/falco/falco_rules.yaml
```

Comment:
As the syscall collection is disabled, we can remove it, same for all settings concerning `modern_pbf`

Reply:
`falco_rules.yaml` provides at least one list, on which the base `k8saudit` rules depend. In our setup I am also including the sandbox rules as there also seems to be a rule dependency on it from base `k8saudit`, can't remember the exact dependency, guess we will find out once ci rule validations runs.

Reply:
rule deps are resolved with #431 , removing the falco_rules.yaml from the list